tutorials/math/Legendre.py

```diff
@@ -28,11 +28,14 @@
 L[0].SetMaximum(1)
 L[0].SetMinimum(-1)
 L[0].SetTitle("Legendre polynomials")
-leg.AddEntry(L[0].Draw(), " L_{0}(x)", "l")
-leg.AddEntry(L[1].Draw("same"), " L_{1}(x)", "l")
-leg.AddEntry(L[2].Draw("same"), " L_{2}(x)", "l")
-leg.AddEntry(L[3].Draw("same"), " L_{3}(x)", "l")
-leg.AddEntry(L[4].Draw("same"), " L_{4}(x)", "l")
-leg.Draw()
+
+for idx, val in enumerate(L):
+    leg.AddEntry(val, " L_{%d}(x)"%(idx), "l")
```

Comment:
Can you use `format` instead of the percent to do the substitution here? It is encouraged for new code.